foreman_maintain.gemspec

```diff
@@ -30,4 +30,5 @@ the Foreman/Satellite up and running."
   s.add_development_dependency 'rake'
   s.add_development_dependency 'rubocop', '0.50.0' # rubocop >= 0.51.0 drops support for ruby 2.0
   s.add_development_dependency 'minitest-stub-const'
+  s.add_development_dependency 'ci_reporter_minitest'
```

Comment:
In Foreman I've migrated to [minitest-reporters](https://github.com/minitest-reporters/minitest-reporters) because it's maintained. The last commit to `ci_reporter_minitest` was in 2014. I've yet to migrate the Smart Proxy repositories, but that's mostly for a lack of time. See https://github.com/theforeman/foreman/commit/b1f03b375e18c8da498ba0944b2565f53441833c for the conversion, which is rather trivial.